src/test/java/nl/dtls/fairdatapoint/service/impl/FairMetaDataServiceImplTest.java

```diff
@@ -318,6 +318,21 @@ public void existenceMetric() throws
                 "http://www.example.com/cat1"));
         assertFalse(metadata.getMetrics().isEmpty());
     }
+    
+    /**
+     * Test existence of access rigghts
+     *
+     * @throws nl.dtls.fairdatapoint.service.FairMetadataServiceException
+     * @throws nl.dtl.fairmetadata4j.io.MetadataException
```

Comment:
Are these possible exceptions that you expect to be thrown by the test under normal execution circumstances? If so, they should be added in the `@Test` annotation attribute [`expected`](https://junit.org/junit4/javadoc/latest/org/junit/Test.html#expected()). If not, I'm not convinced they should be documented in a method level javadoc block. But that's a personal opinion, and up for discussion.